release.config.js

```diff
@@ -5,19 +5,19 @@ module.exports = {
             level: "minor"
         },
         {
-            name: "next",
-            prerelease: true
-        },
-        {
-            name: "lts-*",
+            name: "zowe-v?-lts",
             level: "patch"
         }
+        // {
+        //     name: "next",
+        //     prerelease: true
+        // }
```

Comment:
Thanks for leaving it commented out.
should make things easier when we have a prerelease for v3 😋